Scripts/linux/dtlcommands.py

```diff
@@ -585,25 +590,29 @@ def invoke(self, settings):
         printService = dtlprint.PrintService(settings.quiet, settings.verbose)
         labSvc = dtllabservice.LabService(settings, printService)
 
-        if settings.name is not None:
-            vms = labSvc.getVirtualMachine(settings.subscription, name=settings.name)
-        elif settings.vmid is not None:
-            vms = labSvc.getVirtualMachine(settings.subscription, vmId=settings.vmid)
+        if settings.deletevm and (
+                        settings.name is not None or settings.vmid is not None) and settings.labname is not None:
```

Comment:
I know this was added earlier, but consider calling it vmname instead of name in case we need to use the name of something else later - while this is context specific it will just be a little more intuitive